### PR TITLE
Service: Pulse Oximeter

### PR DESCRIPTION
**Pull request by pelikhan**
This pull request adds a new service definition for JACDAC.

### CODE REVIEW

File: services/pulseoximeter.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+**JACDAC is not suitable for medical devices and should NOT be used in any kind of device to diagnose or treat any medical conditions.**
+
+    identifer: 0x14ad1a5d
+    extends: _sensor
+
+## Registers
+
+    ro oxygen: i8.8 bpm @ reading
+
+The estimated oxygen level in blood. Negative value when the sensor cannot sense the rate.
```
Comment (mmoskal) [inline services/pulseoximeter.md:14]: This value is given as %, so maybe just `u16 /` ?

As in heart rate, 0 could be used to indicate an error (the sensors typically go 75-99%).

Comment (pelikhan) [reply services/pulseoximeter.md]: Actually we have a status code register on every service...

Comment (pelikhan) [reply services/pulseoximeter.md]: oops bpm should be %

Comment (mmoskal) [reply services/pulseoximeter.md]: wait, so you want it to be 100.0 for full saturation or 0xffff ? I guess 100.0 would follow relative humidity

Comment (pelikhan) [reply services/pulseoximeter.md]: i have a % type now.

**Review by mmoskal — APPROVED** [review]

**Pull request merged**